lighthouse-core/test/audits/uses-rel-preload-test.js

```diff
@@ -69,41 +73,63 @@ describe('Performance: uses-rel-preload audit', () => {
         const mainDocumentNodeLocal = nodesByUrl.get(mainDocumentNode.record.url);
         const scriptNodeLocal = nodesByUrl.get(scriptNode.record.url);
         const scriptAddedNodeLocal = nodesByUrl.get(scriptAddedNode.record.url);
+        const scriptSubNodeLocal = nodesByUrl.get(scriptSubNode.record.url);
+        const scriptOtherNodeLocal = nodesByUrl.get(scriptOtherNode.record.url);
 
         const nodeTimings = new Map([
           [rootNodeLocal, {starTime: 0, endTime: 500}],
           [mainDocumentNodeLocal, {startTime: 500, endTime: 1000}],
           [scriptNodeLocal, {startTime: 1000, endTime: 2000}],
           [scriptAddedNodeLocal, {startTime: 2000, endTime: 3250}],
+          [scriptSubNodeLocal, {startTime: 2000, endTime: 3000}],
+          [scriptOtherNodeLocal, {startTime: 2000, endTime: 3500}],
         ]);
 
         if (scriptAddedNodeLocal.getDependencies()[0] === mainDocumentNodeLocal) {
           nodeTimings.set(scriptAddedNodeLocal, {startTime: 1000, endTime: 2000});
         }
 
-        return {timeInMs: 3250, nodeTimings};
+        if (scriptSubNodeLocal.getDependencies()[0] === mainDocumentNodeLocal) {
+          nodeTimings.set(scriptSubNodeLocal, {startTime: 1000, endTime: 2000});
+        }
+
+        if (scriptOtherNodeLocal.getDependencies()[0] === mainDocumentNodeLocal) {
+          nodeTimings.set(scriptOtherNodeLocal, {startTime: 1000, endTime: 2500});
+        }
+
+        return {timeInMs: 3500, nodeTimings};
       },
     };
 
     const mainResource = Object.assign({}, defaultMainResource, {
-      url: 'http://www.example.com',
+      url: 'http://www.example.com:3000',
```

Comment:
is this just to exercise extra hostname checks?

Reply:
yes it's to make sure that origins with a port number are handled correct by the rootDomainMatch